packages/nc-gui/middleware/auth.global.ts

```diff
@@ -137,7 +137,7 @@ async function tryGoogleAuth(api: Api<any>, signIn: Actions['signIn']) {
         data: { token, extra },
       } = await api.instance.post(`/auth/${authProvider}/genTokenByCode${window.location.search}`)
 
-      extraProps = extra
+      extraProps = extra || {}
```

Comment:
The above API doesn't return `extra`. Please double check the logic here.

Reply:
In cloud it's there, that's why we kept.

Reply:
maybe add a comment there.